src/kitsController.ts

```diff
@@ -99,7 +99,8 @@ export class KitsController {
             await KitsController.readUserKits(cmakeProject);
         }
 
-        const folderKitsFiles: string[] = [KitsController._workspaceKitsPath(cmakeProject.folder)].concat(await KitsController.expandAdditionalKitFiles(cmakeProject));
```

Comment:
Is `expandAdditionalKitFiles` used anymore?  If not, please delete it.

Reply:
right, I made the changes to eliminate this function in the first palce.